trie/structural_2.go

```diff
@@ -407,6 +447,10 @@ func (hb *HashBuilder) accountLeaf(length int, fieldSet uint32) error {
 		hb.acc.Balance.Set(balance)
 	}
 	popped := 0
+	if fieldSet&uint32(8) != 0 {
+		copy(hb.acc.CodeHash[:], hb.hashStack[len(hb.hashStack)-popped*33-32:len(hb.hashStack)-popped*33])
```

Comment:
I think we need to use constants for 4,8,32,33

Reply:
Sure